_code-samples/modular-tutorials/credential-manager.js

```diff
@@ -0,0 +1,66 @@
+// *******************************************************
+// *********** Create Credential *************************
+// *******************************************************
+
+async function createCredential() {
+
+  let net = getNet()
+  const client = new xrpl.Client(net)
+  results = `\n\n===Creating Credential===\n\nConnecting to ${getNet()} ...`
+  updateResults()
+  await client.connect()  
+  results = `\n\nConnected.`
+  updateResults()
+
+  try {
+  
+  // Get account wallet from seed
+  const wallet = xrpl.Wallet.fromSeed(accountSeedField.value)
+  
+  // Get subject
+  const subject = subjectField.value
+  
+  // Get credential type - convert string to hex if needed
+  let credentialType = credentialTypeField.value;
+  if (!/^[0-9A-F]+$/i.test(credentialType)) {
```

Comment:
We can use convenience functions from the `@xrplf/isomorphic` package to convert string to hex or hex back to string: `stringToHex`, `hexToString`.

Also mentioned in `xrpl.js` reference:
https://js.xrpl.org/functions/convertStringToHex.html
https://js.xrpl.org/functions/convertHexToString.html